PhysicsTools/PatAlgos/python/tools/coreTools.py

```diff
@@ -42,6 +42,14 @@ def toolCode(self, process):
 
         print '******************* RunOnData *******************'
         removeMCMatching(process, names=names, postfix=postfix, outputModules=outputModules)
+        modStringsToRemove = [
+            'patJetGenJet', 
+            'GenParticle', 
+            'slimmedGenJets', 
+            'tauGenJet', 
+            'patJetFlavourAssociation', 
+            'patJetParton',
+        ]
```

Comment:
It is a simple x in y check. So, yes, anything module label with one of these substrings.